crates/weaver_resolver/src/lib.rs

```diff
@@ -253,6 +255,7 @@ impl SchemaResolver {
     fn load_semconv_from_local_path(
         local_path: PathBuf,
         registry_path_repr: &str,
+        strict_mode: bool,
```

Comment:
Why are we introducing strict mode?

Reply:
Because all the existing versions of SemConv have issues in their examples.

Reply:
Changed strict mode to future mode